packages/apify/src/enqueue_links/click_elements.ts

```diff
@@ -226,24 +242,12 @@ function createInterceptRequestHandler(page, requests) {
     };
 }
 
-/**
- * @param {Page} page
- * @param {PuppeteerRequest} req
- * @return {boolean}
- * @ignore
```

Comment:
keep the @ignore comment, that is for typedoc so it is not part of the API documentation